client/my-sites/site-settings/sitemaps.jsx

```diff
@@ -0,0 +1,196 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes, Component } from 'react';
+import { localize } from 'i18n-calypso';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import SectionHeader from 'components/section-header';
+import JetpackModuleToggle from 'my-sites/site-settings/jetpack-module-toggle';
+import FormFieldset from 'components/forms/form-fieldset';
+import FormSettingExplanation from 'components/forms/form-setting-explanation';
+import InfoPopover from 'components/info-popover';
+import ExternalLink from 'components/external-link';
+import QueryJetpackConnection from 'components/data/query-jetpack-connection';
+import { getSelectedSite, getSelectedSiteId } from 'state/ui/selectors';
+import { getJetpackModule, isActivatingJetpackModule, isJetpackModuleActive } from 'state/selectors';
+import { isJetpackSite } from 'state/sites/selectors';
+
+class Sitemaps extends Component {
+	static defaultProps = {
+		isSavingSettings: false,
+		isRequestingSettings: true,
+	};
+
+	static propTypes = {
+		isSavingSettings: PropTypes.bool,
+		isRequestingSettings: PropTypes.bool,
+	};
+
+	renderSitemapLink( sitemapUrl ) {
+		return (
+			<div key={ sitemapUrl }>
+				<ExternalLink href={ sitemapUrl } icon target="_blank">
+					{ sitemapUrl }
+				</ExternalLink>
+			</div>
+		);
+	}
+
+	renderInfoLink( url ) {
+		const { translate } = this.props;
+
+		return (
+			<div className="sitemaps__info-link-container site-settings__info-link-container">
+				<InfoPopover position={ 'left' }>
+					<ExternalLink href={ url } icon target="_blank">
+						{ translate( 'Learn more about Sitemaps.' ) }
+					</ExternalLink>
+				</InfoPopover>
+			</div>
+		);
+	}
+
+	renderSitemapExplanation() {
+		const { translate } = this.props;
+
+		return (
+			<FormSettingExplanation>
+				{ translate( 'Your sitemap is automatically sent to all major search engines for indexing.' ) }
```

Comment:
I've stuck with the other one for consistency with Jetpack, but I'm not sure if we should switch to this suggested one - it was used in Calypso in the Site Verification Services card. @rickybanister thoughts?